spec/lib/gettext_setup_spec.rb

```diff
@@ -50,8 +52,10 @@
       expect(GettextSetup.default_locale).to eq('en')
       expect(GettextSetup.candidate_locales).to include('en')
       GettextSetup.clear
+      before_locale = ENV['LANG']
       ENV['LANG'] = 'de_DE'
       expect(GettextSetup.candidate_locales).to eq('de_DE,de,en')
+      ENV['LANG'] = before_locale
```

Comment:
Yeah we only need the locale to be `en_US` when we're reading the error messages. I wonder if there's a platform-agnostic way we can change the locale if not through `ENV`.